RGB v2/Led.c

```
@@ -0,0 +1,80 @@
+
+ 
+ /**********************************************************************************************************************
+ *  INCLUDES
+ *********************************************************************************************************************/
+#include "Std_Types.h"
+#include "Led.h"
+#include "Port.h"
+
+/**********************************************************************************************************************
+ *  GLOBAL FUNCTIONS
+ *********************************************************************************************************************/
+/*******************************************************************************
+* Service Name: Led_Init
+* Service ID[hex]: 0x00
+* Sync/Async: Synchronous
+* Reentrancy: Reentrant
+* Parameters (in): None
+* Parameters (inout): None
+* Parameters (out): None
+* Return value: void
+* Description: Function to initialize the connected channel using Port MCAL
+*******************************************************************************/
+void Led_Init(void)
```

Every API should return an error state